.github/workflows/scheduled-run.yml

```diff
@@ -0,0 +1,16 @@
+name: scheduled-run
+
+on: push
+  # schedule:
+  #   - cron:  '*/15 * * * *'
+
+jobs:
+  scheduled-run:
+    runs-on: ubuntu-latest
+    container: quay.io/enarx/fedora
```

Comment:
@npmcallum: We probably don't want to use our container for this. It is rather large. If `ubuntu-latest` has `PyGithub`, we probably don't need a container at all.